### DISCUSSION TITLE
Welcome to myterms.fyi Discussions!

### DISCUSSION THREAD

**Discussion by solvaholic** [solvaholic/myterms.fyi#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your interest in this community.

**Comment by solvaholic** [comment]
:wave: Hiyo! I'm @solvaholic, thank you for stopping by :bow:

_myterms.fyi_ is the scratch I envision to this itch:

> I set up surveillance cameras. How can I communicate to customers, visitors, and passers by what I do with the images those cameras collect?
> 
> How can I let them know it's important to me that I - and the camera, monitoring, or storage providers I involve - respect their privacy and rights?

**Comment by letsnotsay** [comment]
I have long used incorporation by reference to point to some repository of specific terms I like to propound, usually in the context of replies to dunning letters seeking to overbill for goods or services, like the so-called "surprise" medical billings in the news of late, or the vicious terms of service used on cell phone carrier agreements which claim to impose arbitration.

My theory is that you can agree to anything, and if your computer offers me a deal, I can counteroffer back, otherwise its a contract of adhesion and likely involves fraudulent concealment, since it is widely known no one reads burdensome documents offered in terms of service except lawyers and those who bedevil them. I fall into the latter category.

I propose your domain myterms.fyi could host personal terms of service which could be appended to signatures manual or electronic via a web shortening/redirection method and then be used to indicate that the signing party has their own terms and that they are inserting them into the offered contract by posting them at the end of their signature. Since the people offering these traps seldom expect to be entrapped the same way, they will generally perform under the terms inserted, and if one of those terms says performance indicates acceptance of the counter terms well you see the fun begins.... Thoughts? How would one obtain a folder under your domain for such use?

**Comment by solvaholic** [comment]
Hi @letsnotsay, thank you so much for commenting here! :bow:

> I propose your domain myterms.fyi could host personal terms of service which could be appended to signatures manual or electronic via a web shortening/redirection method and then be used to indicate that the signing party has their own terms and that they are inserting them into the offered contract by posting them at the end of their signature.

I love it! You could link your terms to anything that way.

While I'm uninterested in bedeviling lawyers, I am interested in making it easier for people to notice the terms they're agreeing to.

> myterms.fyi could host personal terms of service

Hosting the terms themselves in a repository you don't control will complicate your maintenance of them. Likewise, controlling every aspect of hosting is a lot of work.

Would it meet your needs to host your terms with a version control system of your choice, and then link to them from myterms.fyi?

Why put terms in version control? I like that approach because it:
- maintains history,
- provides in-context options for proposing, evaluating, and documenting changes, and
- provides means to notice when terms have been altered.

I'm familiar with GitHub, and haven't evaluated others, but I imagine most version control service providers offer something like GitHub's [profile READMEs](https://docs.github.com/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme), [gists](https://docs.github.com/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists), and [repositories](https://docs.github.com/repositories).

> How would one obtain a folder under [myterms.fyi] for such use?

:joy: A fine question indeed. I started thinking about that over in https://github.com/solvaholic/myterms.fyi/issues/18, please join in or follow along there.

**Reply by letsnotsay** [reply]
Thank you I will go look at the suggested postings and see how that works, I've never been forced to use an RCS for anything, and so never got comfortable with them. I guess I am wildly un-disciplined for someone who cut their teeth on machines that booted from paper tape LOL. Back then we just tossed the code in a pile and wrote something descriptive with a sharpie on its container. The whole check it in check it out thing is something I probably something I should pick-up before I croak... its a wonder some passing middle manager didn't corner me decades ago. 

Let me see if I can figure this git thing out a bit more, everything I know about it to date involves getting something out to install on a machine. My ultimate hope is something short and sweet that can go onto a normal signature pointing to the tripewire of terms of service LOL.

**Reply by solvaholic** [reply]
> Let me see if I can figure this git thing out a bit more, everything I know about it to date involves getting something out to install on a machine.

Aye, it can become _very_ involved. However, GitHub and GitLab (and, I assume, others) will let you do file creation and editing in a browser. For something as straightforward as publishing one text or markdown document, [gists](https://docs.github.com/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) may be the way to go.

If you'd like to publish the document with your GitHub account, you could simply follow this link to create a gist, add your content, and save it: https://gist.github.com

And, of course, you could host your terms anywhere you like. My own preference for onymity and transparency - and the extent to which I use GitHub - makes platforms like GitHub, GitLab, and Bitbucket natural and obvious choices for hosting text and code. If you're not already using them to manage code or text documents, I imagine other options may better suit your needs.
